coretex/cli/project.py

```diff
@@ -68,5 +68,4 @@ def select(name: Optional[str], id: Optional[int]) -> None:
 def project() -> None:
     pass
 
-
```

Comment:
Why was this newline removed?

Reply:
On that spot, there were two empty lines, while everywhere else, there was only one, so I deleted the extra one.

Reply:
Top-level spacing in python uses 2 newlines not one